packages/libs/client/src/client/client.ts

```diff
@@ -238,20 +253,23 @@ export interface ICreateClient {
    *
    * Useful when you are working with a single network and chainId.
    * @param hostUrl - the URL to use in the client
+   * @param defaults - default options for the client it includes confirmationDepth that is used for polling
    */
-  (hostUrl: string): IClient;
+  (hostUrl: string, defaults?: { confirmationDepth?: Milliseconds }): IClient;
```

Comment:
Is this a mistake? Should this be number?

Reply:
yes will fix it